src/Services/VitalsService.php

```diff
@@ -195,8 +195,7 @@ public function search($search, $isAndCondition = true)
                 $recordsById[$row['form_id']] = $record;
             }
             if (!empty($details)) {
-                $existingRecord = &$recordsById[$row['form_id']];
-                $existingRecord["details"][$details['vitals_column']] = $details;
+                $recordsById[$row['form_id']]["details"][$details['vitals_column']] = $details;
```

Comment:
@adunsulag ,
This was an odd bug where it would duplicate read out on 1st 2 vital signs if details (ie. high, low, etc.) were set in any of the vitals signs. I basically just bypassed the array reference mechanism and the bug went away :)